odc/stats/plugins/gm.py

```diff
@@ -20,10 +20,9 @@ def __init__(
         self,
         bands: Tuple[str, ...],
         mask_band: str,
-        cloud_classes: Tuple[str, ...],
         nodata_classes: Optional[Tuple[str, ...]] = None,
-        filters: Optional[Iterable[Tuple[str, int]]] = None,
-        basis_band=None,
+        cloud_filters: Optional[Dict[str, Iterable[Tuple[str, int]]]] = None,
```

Comment:
In this case `cloud_filters` is not `optional` anymore, only the `value` of `Dict` (dilation strategy) is. Otherwise, if there is no `dilation`, there won't be a cloud/shadow mask at all.